CONTRIBUTING.md

```diff
@@ -26,10 +32,26 @@ The last option is needed because our guides feature code examples. **markdown-s
 
 For now it'll simply report all errors and you can then deal with actually spelling errors directly.
 
-We use [markdownlint](https://github.com/mivok/markdownlint) to check the files have a consistent style as part of a CI process. We check contributions using `mdl --style md_style.rb $PWD`. We recommend you install this and run it before pushing your commits.
+We use [markdownlint](https://github.com/mivok/markdownlint) to check the files have a consistent style as part of a CI process. We check contributions using `mdl --style md_style.rb $PWD`. We recommend you install this and run it before requesting feedback.
+
+## When you're finished
+
+Make sure everything is pushed, then add the **ready for review** label to the PR and ensure you are assigned. If you have access to it announce your PR in our [Slack](https://defra-digital.slack.com/) [#dst-guides](https://defra-digital.slack.com/messages/dst-guides/) channel.
+
+If you don't add a comment mentioning either **@cruikshanks** or **@bensagar-ea**.
```

Comment:
I think we should just say that we announce it on Slack and that everyone should have access to that.  Especially as we've decided to use it as a common communication tool for the software development community.